### ISSUE TITLE
Library version has not been updated since 0.5.0

### ISSUE THREAD

**Issue by sjoelund** [mtiller/ModelicaBook#436]
Is there a reason that ModelicaByExample has not been updated for any of the tagged releases of the book?

It might be OK that v0.5.0 - v0.5.3 are the same ModelicaByExample version since they were not changed. But v0.6.0 seems to have some changes (but is still called 0.5.0).

**Comment by mtiller** [comment]
Martin, I'm not 100% sure what we are talking about.  The whole repo is tagged with different versions.  Are you talking about the Version annotation inside the library?!?

**Comment by beutlich** [comment]
Yes, that version should match the tagged repository/book version (0.6.0), but it does not:

https://github.com/mtiller/ModelicaBook/blob/6a23eb9d5aa8c1187a67ab833d20e265798a329b/ModelicaByExample/package.mo#L3

**Comment by beutlich** [comment]
By the way, would be appreciated to migrate from the obsolete MSL version 3.2.2, too. See my other open issues #406 and #439 for it.
